### DISCUSSION TITLE
Referencing column are incompatible when migrate db

### DISCUSSION THREAD

**Discussion by Armando99Rdz** [adonisjs/core#3185] [Help]
Hello!, I'm new here. I have a issue when referencing columns. I just execute the command `migration:run` console display: 

![Captura realizada el 2021-09-16 14 47 38](https://user-images.githubusercontent.com/59629345/133676415-037d7b96-55b7-43e9-b825-cf5237dc809d.png)

The rest of migrations without foreign keys definitions works perfectly.

## table A
![Captura realizada el 2021-09-16 14 31 46](https://user-images.githubusercontent.com/59629345/133674502-06f65e01-a763-4937-8fb6-b47d0a6214d4.png)

## table B
![Captura realizada el 2021-09-16 14 27 16](https://user-images.githubusercontent.com/59629345/133674599-31cec6f5-3170-4fb9-be05-6c4d69a87d31.png)

Do I do something wrong? :tired_face: 

I've tried adding `unsigned()`, specifying and removing `charset` even `collate`  but not works.

## context
`node -v`: v14.17.6
`npm -v`: 6.14.15 
`mysql --version`: 8.0.26-0ubuntu0.20.04.2 for Linux on x86_64 (Ubuntu)

"@adonisjs/core": "^5.3.4",
 "@adonisjs/lucid": "^16.0.2",

**Comment by thetutlage** [comment]
Looks like you have tried many things like removing `unsigned`, removing `charset` and so on. Maybe a specific combination was failing every time.

- I suggest you to remove any sort of charset config from all the migrations
- Then add `unsigned` to the FK

and then run migrations
